addons/package_hierarchy/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "version": "11.0",
     "summary": "Inventory, Logistics, Warehousing",
     "description": "Add the ability for multi-level packages back to Odoo",
-    "depends": ["stock"],
+    "depends": ["stock","udes_common"],
```

Comment:
Nit: I think should have space after comma

Reply:
Done